### DISCUSSION TITLE
How to add new string to errorformat in init.lua

### DISCUSSION THREAD

**Discussion by fabz-fr** [neovim/neovim#28151] [Q&A]
Hello everyone,

I want to jump through Python backtrace in neovim quickfix list. 
To do that, I added the following ex-command `:set errorformat+=%.%#File\ \"%f\"\\,\ line\ %l%.%#` and this command works well.
What I would like now is to add, this command in my init.lua, But when I add the previous line, neovim raises errors when I populate the quickfix List.
This is the command I added to my init.lua : `vim.opt.errorformat:append { "%.%#File \"%f\"\\, line %l%.%#" }`.

Here is the error when I populate the quickfix list : 
`Error detected while processing function AsyncRun_Job_OnTimer[12]..<SNR>24_AsyncRun_Job_Update[40]..<SNR>24_AppendText: line   33: E377: Invalid %- in format string`

Do you know how to append a new string to errorFormat ? 

Thank you

**Comment by zeertzjq** [comment]
This seems to be a bug in `vim.opt`. It treats escaped commas as separators.

**Reply by fabz-fr** [reply]
I think It is a normal behavior for errorformat to handle commas as a special characters.

The thing is, even `vim.opt.errorformat:append('foo%f')` does not work which is weird

**Comment by gibfahn** [comment]
Looks like this is actually an issue in neovim, and someone filed it a while back: https://github.com/neovim/neovim/issues/29061
